app/build.gradle

```diff
@@ -179,18 +179,18 @@ dependencies {
     androidTestImplementation 'app.cash.turbine:turbine:0.12.1'
 
     //ViewPager
-    implementation 'com.google.accompanist:accompanist-pager:0.28.0'
+    implementation "com.google.accompanist:accompanist-pager:$accompanist_version"
 
     //Exoplayer
     api 'com.google.android.exoplayer:exoplayer-core:2.18.2'
     api 'com.google.android.exoplayer:exoplayer-ui:2.18.2'
     api 'com.google.android.exoplayer:extension-mediasession:2.18.2'
 
     //Room db
-    implementation "androidx.room:room-runtime:2.5.0"
-    kapt "androidx.room:room-compiler:2.5.0"
-    implementation "androidx.room:room-ktx:2.5.0"
+    implementation "androidx.room:room-runtime:$room_version"       // DON'T UPDATE TO 2.5.0
```

Comment:
I really dislike that the bump caused runtime errors rather than compile time. 
For future reference, we should bump to 2.5.0 following the guide here https://developer.android.com/jetpack/androidx/releases/room#version_250_3